scripts/build/build.mk

```diff
@@ -160,7 +160,7 @@ hubl:
 
 #? mocks: Generate mock file
 mocks: $(MOCKS_DIR)
-	@go install github.com/golang/mock/mockgen@v1.6.0
+	@go install go.uber.org/mock/mockgen@v0.5.0
```

Comment:
hum, main uses go.uber.org but release/v0.52.x should still use github.com/golang/mock/mockgen. Otherwise you have a bunch of tests to update.